api/src/main/java/io/minio/MinioClient.java

```diff
@@ -671,15 +671,7 @@ public MediaType contentType() {
 
```

Comment:
This is not a right fix.
`requestBody` is created only if `body != null`.  As per `createRequest()`, `body` is any object or `null`.  If `body` is non-null and other than `(body instanceof InputStream || body instanceof RandomAccessFile || body instanceof byte[])`, we won't need to compute md5sum and need to send `Transfer-Encoding: chunked`